GPURigidRegistration.cpp

```diff
@@ -100,21 +107,15 @@ int main(int argc, char* argv[])
     outputTransform->SetMatrix(matrix);
     outputTransform->SetOffset(offset);
     
-    // temporary testing
-    for( size_t i = 0; i < 4; i++ )
-    {
-        for( size_t j = 0; j < 4; j++ )
-        {
-            std::cout << transform->GetMatrix()->GetElement(i, j) << "\t";
-        }
-        std::cout << std::endl;
-    }
+    std::string outputTransformFile = "outputTransform.h5";
+    if( argc > 3 )
+        outputTransformFile = argv[3];
 
-    std::cout << "Write transform in " << argv[4] << std::endl;
+    std::cout << "Write transform in " << outputTransformFile << std::endl;
     using TransformWriterType = itk::TransformFileWriterTemplate<ScalarType>;
     TransformWriterType::Pointer writer = TransformWriterType::New();
-    writer->SetInput(outputTransform);
-    writer->SetFileName(argv[4]);
+    writer->SetInput(outputTransform->GetInverseTransform());

```

Comment:
I don't like this. The transform is inverted... so I had to invert it again to write it in the correct state in a .h5 or .tfm file.